src/gtk/webview_webkit2.cpp

```diff
@@ -1279,6 +1244,10 @@ bool wxWebViewWebKit::RunScript(const wxString& javascript, wxString* output)
 void wxWebViewWebKit::RegisterHandler(wxSharedPtr<wxWebViewHandler> handler)
 {
     m_handlerList.push_back(handler);
+    WebKitWebContext* context = webkit_web_context_get_default();
+    webkit_web_context_register_uri_scheme(context, handler->GetName(),
```

Comment:
Missing `utf8_str()` here too, see the same post for the error details.

Reply:
Fixed.